examples/default_categories.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# coding=utf-8
+"""
+Simple example demonstrating basic CommandSet usage.
+"""
+
+import cmd2
+from cmd2 import CommandSet, with_default_category
+
+
+@with_default_category('Default Category')
+class MyBaseCommandSet(CommandSet):
+    pass
+
+
+class ChildInheritsParentCategories(MyBaseCommandSet):
+    def do_hello(self, _: cmd2.Statement):
+        self._cmd.poutput('Hello')
+
+    def do_world(self, _: cmd2.Statement):
+        self._cmd.poutput('World')
+
+
+@with_default_category('Non-Heritable Category', heritable=False)
+class ChildOverridesParentCategoriesNoHeritable(MyBaseCommandSet):
+    def do_goodbye(self, _: cmd2.Statement):
+        self._cmd.poutput('Goodbye')
+
+
+class GrandchildInheritsNoHeritable(ChildOverridesParentCategoriesNoHeritable):
```

Comment:
Is `GrandchildInheritsNoHeritable` the correct name for this class? Doesn't it inherit the category the default category from `MyBaseCommandSet`?